src/opus/imported_decode.rs

```diff
@@ -24,12 +23,12 @@ pub type ec_window = opus_uint32;
 pub struct ec_dec<R> where R: std::io::Read {
     pub inp: R,
     pub end_window: ec_window,
-    pub nend_bits: libc::c_int,
-    pub nbits_total: libc::c_int,
+    pub nend_bits: std::os::raw::c_int,
```

Comment:
Could you take the opportunity to replace all these `std::os::raw::*` with the pub type defined above?

Reply:
I don't get it

Reply:
Nevermind, I'll do it in a followup.